clearwater-infrastructure/usr/bin/clearwater-aio-install

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
So this script is part of clearwater-infrastructure... isn't that a bit chicken-and-egg?